frontend/src/types.ts

```diff
@@ -1154,13 +1154,16 @@ export interface LegacyRecordingFilters {
     operand?: FilterLogicalOperator
 }
 
+export type SortDirection = 'ASC' | 'DESC'
+
 export interface RecordingUniversalFilters {
     date_from?: string | null
     date_to?: string | null
     duration: RecordingDurationFilter[]
     filter_test_accounts?: boolean
     filter_group: UniversalFiltersGroup
     order?: RecordingsQuery['order']
+    direction?: SortDirection
```

Comment:
🤔 so this makes the API more powerful than the UI
e.g. I can sort for least active
that's not a terrible thing
but we could copy the DRF/Django approach and accept `start_time` and `-start_time` and then pick that out when we construct the query

i'm not actually sure which I prefer

Reply:
Yeah, UI can be just as powerful if we split the direction logic into a new button. Keeping order + direction felt cleaner and more type-safe to me, but I like the DRF-style too. Also not sure which I prefer

Reply:
Thinking about it, even if we add a direction button in the UI, DRF style would still work fine… Looks like a solid way to go

Reply:
yeah, we've talked a few times about wanting to have more in the API than in the UI (so folk can always self-serve)

i think here no button in the UI is fine

and we can allow `start_time` and `-start_time` (or whatevs typed from memory 😅 )

that way we can avoid a lot of the schema changes and just check as we generate the order by in the query

i'm on PTO next week but @veryayskiy can help keep things moving if we don't nail this today, really appreciate your patience with backwards and forwards here! API/schema always slightly more painful to change since we're stuck with it once released

Reply:
Oh, no worries at all. Really enjoying the back and forth. It’s been super nice contributing to PostHog! Probably won’t get to it before the weekend, but sounds like a solid plan. Enjoy the PTO and thanks for looping in @veryayskiy 🙌